optimum/exporters/openvino/convert.py

```diff
@@ -389,7 +389,10 @@ def ts_patched_forward(*args, **kwargs):
                 if patch_16bit_model:
                     from openvino.frontend.pytorch.patch_model import __make_16bit_traceable
 
+                    # frontend may riases confusing warnings about modules already patched if model splitted on several parts
+                    logging.disable(logging.WARNING)
                     __make_16bit_traceable(model)
+                    logging.disable(logging.NOTSET)
```

Comment:
What if some levels were already disabled? After this they'll become enabled back.

Reply:
good question on which I can not find answer. Also looks like warning was replaced by debug info, so in 2024.5 there is no need to have disabling https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/frontend/pytorch/patch_model.py#L79